code/controllers/subsystems/supply.dm

```diff
@@ -69,7 +69,7 @@ SUBSYSTEM_DEF(supply)
 
 		msg += "[export_text]<br>"
 		points += E.total_cost
-		E.export_end()
+		exports.Cut()
```

Comment:
You did that in a loop, and you needed to do it after the loop is done. It would break the loop otherwise.

Reply:
Just remove the tab before `exports.Cut()` to bring it in line with the `for` loop before it so that it's not part of the loop and this will work. Tested on a local server and it works wonderfully.